nightwatch.conf.js

```diff
@@ -0,0 +1,266 @@
+        start_process: true,
+        server_path: '/usr/bin/safaridriver'
+      }
+    },
+
+    firefox: {
+      desiredCapabilities: {
+        browserName: 'firefox',
+        alwaysMatch: {
+          // Enable this if you encounter unexpected SSL certificate errors in Firefox
+          // acceptInsecureCerts: true,
+          'moz:firefoxOptions': {
+            args: [
+              // '-headless',
+              // '-verbose'
+            ]
+          }
+        }
+      },
+      webdriver: {
+        start_process: true,
+        port: 4444,
+        server_path: Services.geckodriver ? Services.geckodriver.path : '',
+        cli_args: [
+          // very verbose geckodriver logs
+          // '-vv'
+        ]
+      }
+    },
+
+    chrome: {
+      desiredCapabilities: {
+        browserName: 'chrome',
+        chromeOptions: {
+          // This tells Chromedriver to run using the legacy JSONWire protocol (not required in Chrome 78)
+          // w3c: false,
+          // More info on Chromedriver: https://sites.google.com/a/chromium.org/chromedriver/
+          args: [
+            //'--no-sandbox',
+            //'--ignore-certificate-errors',
+            //'--allow-insecure-localhost',
+            //'--headless'
+          ]
+        }
+      },
+
+      webdriver: {
+        start_process: true,
+        port: 9515,
+        server_path: Services.chromedriver ? Services.chromedriver.path : '',
+        cli_args: [
+          // --verbose
+        ]
+      }
+    },
+
+    //////////////////////////////////////////////////////////////////////////////////
+    // Configuration for when using the browserstack.com cloud service               |
+    //                                                                               |
+    // Please set the username and access key by setting the environment variables:  |
+    // - BROWSERSTACK_USER                                                           |
+    // - BROWSERSTACK_KEY                                                            |
+    // .env files are supported                                                      |
+    //////////////////////////////////////////////////////////////////////////////////
+    browserstack: {
+      selenium: {
+        host: 'hub-cloud.browserstack.com',
+        port: 443
+      },
+      // More info on configuring capabilities can be found on:
+      // https://www.browserstack.com/automate/capabilities?tag=selenium-4
+      desiredCapabilities: {
+        'bstack:options': {
+          userName: '${BROWSERSTACK_USER}',
+          accessKey: '${BROWSERSTACK_KEY}'
+        }
+      },
+
+      disable_error_log: true,
+      webdriver: {
+        keep_alive: true,
+        start_process: false
+      }
+    },
+
+    'browserstack.local': {
+      extends: 'browserstack',
+      desiredCapabilities: {
+        'browserstack.local': 'true'
+      }
+    },
+
+    'browserstack.local_chrome': {
+      extends: 'browserstack',
```

Comment:
does it work to extend `browserstack.local` here?

Reply:
Nope, multiple `extends` does not seem to work